content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -28,11 +30,11 @@ The fields passed to the `datasource` block are:
 
 ### Connection URL
 
-Prisma follows the [official PostgreSQLl format](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) for connection URLs. Here's an overview of the components needed for a PostgreSQL connection URL:
+Prisma is based on the [official PostgreSQLl format](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) for connection URLs, but does not support all arguments and includes additional arguments such as `schema`. Here's an overview of the components needed for a PostgreSQL connection URL:
```

Comment:
"Prisma is based on ..." might not be correct here - the connection URL format we are using is.